model/account.go

```diff
@@ -30,11 +30,14 @@ type InternalAccount struct {
 	ProjectChildID
 	Index             int
 	Address           Address
-	DraftCode         string
-	DeployedCode      string
 	DeployedContracts []string
 	marshalledState   string
 	unmarshalledState AccountState
+
+	// for migrating code inside
+	// Account struct to seperate Contract struct
+	DraftCode    string
```

Comment:
See comment in `migrate.go` about removing these.

Reply:
yes